cmd/eqpgsvc/cmd/root.go

```diff
@@ -132,6 +153,9 @@ func init() {
 	pflags.StringVar(&dbPass, "dbpwd", "", "Database password. Overrides PGPASSWORD environment.")
 	pflags.IntVar(&attempts, "attempts", 10, "Connection attempts, separated by 5-second pauses, before dying due to lack of backend connection.")
 	pflags.IntVar(&maxSize, "max_size_mb", 10, "Maximum server message size (send and receive) in megabytes. If larger than 4MB, you must also set your gRPC client max size to take advantage of this.")
+	pflags.StringVar(&authzStrategy, "authz", "none", "Strategy to use for authorization. Default is no authorization, everything allowed by every rquester.")
```

Comment:
I know that you mentioned refactoring things so that the various storage versions of EQ don't have to reimplement all the shared portions. This appears to be one of those areas :) Just making sure to keep eyes on the long term adjustments to the structure. No changes required now, obviously.

Reply:
Yes indeed! Thanks for eyes on this.